components/03-modules/mega-menu/mega-menu.js

```diff
@@ -0,0 +1,74 @@
+class MegaMenu {
+  constructor(element) {
+    this.menu = element;
+    this.focusable = 'a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), object, embed, *[tabindex], *[contenteditable]';
+    this.focusableChildren = Array.from(this.menu.querySelectorAll(this.focusable));
+    this.firstLevelLinks = Array.from(this.menu.querySelectorAll('.mega-menu__item > .mega-menu__link'));
+    this.content = document.getElementById('contentarea');
```

Comment:
Is this element with id `contentarea` located somewhere?

Reply:
yes, in the theme we have a link, It will probably change after implementing of skip-nav link, but I'll keep it updated.